<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Etage</string>
    </attribute>
    <widget class="QPushButton" name="ButtonExcel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Code de l' Etage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de pieces</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="spCode">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="cbNbPieces">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Aucune piece</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Une piece</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 pieces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 pieces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 pieces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 pieces</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6 pieces</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="img">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="leRecherche">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouterEtages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerEtages">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabEtages">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>341</width>
       <height>301</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonModifererEtages">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTri">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numéro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matiére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Déscription</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Chambre</string>
    </attribute>
    <widget class="QTableView" name="tabChambres">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>341</width>
       <height>301</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ref Chambre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Occupant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LbSuperficie">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Superficie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Statut</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de lits</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sprefch">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="LbStatut">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Occupee</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Libre</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="cbOcc">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LbNbLits">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonModifererChambres">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTriChambres">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numéro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matiére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Déscription</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonAjouterChambres">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leRechercheChambres">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonExcelChambres">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PDFChambre">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>360</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerChambres">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
